docs/guide/getting-started.md

```diff
@@ -152,8 +152,8 @@ Add `asdf.elv` to your `~/.elvish/rc.elv` with:
 
 ```shell:no-line-numbers
 mkdir -p ~/.elvish/lib; ln -s ~/.asdf/asdf.elv ~/.elvish/lib/asdf.elv
-echo "\n"'use asdf _asdf; fn asdf [@args]{_asdf:asdf $@args}' >> ~/.elvish/rc.elv
-echo "\n"'edit:completion:arg-completer[asdf] = $_asdf:arg-completer~' >> ~/.elvish/rc.elv
+echo "\n"'use asdf _asdf; fn asdf {|@args| _asdf:asdf $@args}' >> ~/.elvish/rc.elv
```

Comment:
In v0.17.0, the [Elvish RC file](https://elv.sh/ref/command.html#rc-file) location was changed.

> - If the legacy `~/.elvish/rc.elv` exists, it is used (this will be ignored in a future version).
> - Otherwise:
>   - On UNIX (including macOS), `$XDG_CONFIG_HOME/elvish/rc.elv` is used, defaulting to `~/.config/elvish/rc.elv` if `$XDG_CONFIG_HOME` is unset or empty.
>   - On Windows, `%AppData%\elvish\rc.elv` is used.

The docs should be updated to reflect these changes, but I'm not sure what to replace `~/.elvish/rc.elv` with as the RC file varies on different systems and configurations. Maybe it could be replaced with `~/.config/elvish/rc.elv` and the RC file docs could be linked above the script?

Reply:
I changed everything to `~/.config/elvish`. What to you think?

Reply:
There are also some references to the old config location here:

https://github.com/asdf-vm/asdf/blob/2a538fe63144c3d0710df23738c1f06363b10fa6/test/asdf_elvish.bats#L5-L13

Reply:
Oh thanks, missed that. It's done.